src/input_xml.F90

```diff
@@ -124,9 +124,9 @@ subroutine read_input_xml() bind(C)
     type(VectorReal), allocatable :: sab_temps(:) ! List of T to read for each S(a,b)
 
     call read_settings_xml()
+    call read_geometry_xml()
```

Comment:
Nope! A lot of the in-memory reading of materials was so that we could avoid this change. I would've felt badly leaving this in here. Nice catch. Thank you.